source/org/zfin/mutant/repository/HibernateMutantRepository.java

```diff
@@ -1800,7 +1800,7 @@ public boolean createFish(Fish fish, Publication publication) {
         Fish existingFish = getFishByGenoStr(fish);
         boolean newFishCreated = false;
         if (existingFish != null) {
-            fish = existingFish;
+            fish.setFishProperties(existingFish);
```

Comment:
This approach feels clunky... 
I'd rather re-fetch the record through Hibernate from the database (see my PR).